### ISSUE TITLE
Support new API Gateway HTTP APIs

### ISSUE THREAD

**Issue by alessandrojcm** [erm/mangum#95]
Hi there, I'm getting the following error when my app executes in API Gateway (paths shortened for brevity):

```bash
Traceback (most recent call last):
  File "/virtualenv/lib/python3.8/site-packages/gunicorn/workers/sync.py", line 134, in handle
    self.handle_request(listener, req, client, addr)
  File "/virtualenv/lib/python3.8/site-packages/gunicorn/workers/sync.py", line 175, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/virtualenv/lib/python3.8/site-packages/mangum/adapter.py", line 61, in __call__
    raise exc
  File "/virtualenv/lib/python3.8/site-packages/mangum/adapter.py", line 59, in __call__
    response = self.handler(event, context)
  File "/virtualenv/lib/python3.8/site-packages/mangum/adapter.py", line 76, in handler
    response = self.handle_ws(event, context)
  File /virtualenv/lib/python3.8/site-packages/mangum/adapter.py", line 130, in handle_ws
    request_context = event["requestContext"]
KeyError: 'requestContext'

```
Here is my FastApi app code:

```python
from fastapi import FastAPI

from app.api import router


def get_application(config) -> FastAPI:
    application = FastAPI(
        title=config.PROJECT_NAME, debug=config.DEBUG, version=config.VERSION
    )
    application.include_router(router, prefix=config.API_PREFIX)

    return application
```

And here is the Mangum adapter:
```python
from mangum import Mangum

from .app import get_application
from .core import config

handler = Mangum(app=get_application(config), enable_lifespan=False)
```

Also tried running locally with `gunicorn`, same error. If I run the app normally with `uvicorn` it works fine.

I'm using the latest version of FastAPI (0.54.1) and Python 3.8 on a Manjaro Linux machine. 

So, according to the examples I've read, that looks like pretty standard stuff; I looked online but couldn't find anything (and the issues posted here seems like they are for old versions), am I missing something?

**Comment by alessandrojcm** [comment]
Oh it looks like PR #94 addresses this, what's the status on that?

**Comment by erm** [comment]
I haven't had a chance to review that PR properly, but there are a few issues that will need to be resolved before it is mergeable. I'll try to look into it over the weekend.

**Comment by alessandrojcm** [comment]
Cool thanks

**Comment by erm** [comment]
I've merged the PR for this into `master`, but there are a few things I want to do before the next release (probably this weekend). I'll leave this open until then.

**Comment by alessandrojcm** [comment]
Great! Thanks for the update @erm

**Comment by erm** [comment]
Released in 0.8.0.
